scripts/oe-env-prepare.ps1

```diff
@@ -0,0 +1,446 @@
+    if(Test-Path -Path $shellcheckDest) {
+        Remove-Item -Path $shellcheckDest -Force -Recurse
+    }
+    New-Item -ItemType Directory -Path $shellcheckDest
+    Move-Item "$PACKAGES_DIRECTORY\shellcheck.exe" $shellcheckDest -Force
+}
+
+function Install-DCAPDrivers {
+    Install-Tool -InstallerPath $PACKAGES["dcap"]["local_file"] `
+                 -ArgumentList "/auto $PACKAGES_DIRECTORY\Intel_SGX_DCAP"
+
+    $drivers = @{
+        'sgx_base_dev' = @{
+            'zip_path' = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\LC_driver_WinServer2016\Signed_*.zip"
+            'location' = 'root\SgxLCDevice'
+        }
+        'sgx_dcap_dev' = @{
+            'zip_path' = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\DCAP_INF\WinServer2016\Signed_*.zip"
+            'location' = 'root\SgxLCDevice_DCAP'
+        }
+    }
+    foreach($driver in $drivers.Keys) {
+        $zip = Get-Item $drivers[$driver]['zip_path']
+        if(!$zip) {
+            Throw "Cannot find the zile file with $driver"
+        }
+        if($zip.Count -gt 1) {
+            $zip
+            Throw "Multiple driver zip files found"
+        }
+        New-Item -ItemType Directory -Force -Path "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\$driver"
+        Expand-Archive -Path $zip -DestinationPath "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\$driver" -Force
+        $inf = Get-Item "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\$driver\drivers\*\$driver.inf"
+        if(!$inf) {
+            Throw "Cannot find $driver.inf file"
+        }
+        if($inf.Count -gt 1) {
+            $inf
+            Throw "Multiple $driver.inf files found"
+        }
+        & "$PACKAGES_DIRECTORY\devcon.exe" install "$($inf.FullName)" $drivers[$driver]['location']
+        if($LASTEXITCODE) {
+            Throw "Failed to install $driver driver"
+        }
+    }
+    $OE_NUGET_DIR = "C:\openenclave\prereqs\nuget"
+    $TEMP_NUGET_DIR = "$PACKAGES_DIRECTORY\Azure_DCAP_Client_nupkg"
+    New-Directory -Path $OE_NUGET_DIR
+    New-Directory -Path $TEMP_NUGET_DIR
+    Copy-Item "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\nupkg\*\*.nupkg" -Destination $TEMP_NUGET_DIR -Force
+    Copy-Item $PACKAGES['azure_dcap_client_nupkg']['local_file'] -Destination $TEMP_NUGET_DIR -Force
+    & "$PACKAGES_DIRECTORY\nuget.exe" install 'EnclaveCommonAPI' -Source "$TEMP_NUGET_DIR" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+    & "$PACKAGES_DIRECTORY\nuget.exe" install 'DCAP_Components' -Source "$TEMP_NUGET_DIR" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+    & "$PACKAGES_DIRECTORY\nuget.exe" install 'Microsoft.Azure.DCAP.Client' -Source "$TEMP_NUGET_DIR;nuget.org" -OutputDirectory "$OE_NUGET_DIR" -ExcludeVersion
+
+    New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters" -Name "SGX_Launch_Config_Optin" -Value "1" -PropertyType DWORD -Force
+
+    Add-ToSystemPath -Path 'C:\Program Files\shellcheck'
```

Comment:
All the below `Add-ToSytemPath` shouldn't be in the `Install-DCAPDrivers` function as they are not related to `DCAPDrivers`.

Please move them to their own functions.